### ISSUE TITLE
amp-selector: toggle action does not layout children properly

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#20640]
From @devignerforhire:

We're experiencing the same issue as described above, using amp-accordion and amp-selector for our wip style guide (https://winefolly.staging.wpengine.com/style-guide/). If I can't get this working I'm going to have to remove the tabs/accordions and have everything stacked, which isn't ideal. TIA :)

The issue seems limited to `toggle` action, tapping selector does layout properly.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by wassgha** [comment]
@devignerforhire could you please provide more information about this issue? I don't see it occurring on the provided link.

**Comment by SimonHogstromRonbol** [comment]
@wassgha i have a real world example here, on https://www.capidea.dk/virksomheder where the clicking the thumbnail logos only renders the amp-img inside the amp-selector the first time a thumbnail is clicked, not subsequent times:
https://www.capidea.dk/virksomheder

i've made a codepen that reproduces this issue:
https://codepen.io/sunturion/pen/QRxMoa

the code in the pen is taken directly from https://amp.dev/documentation/examples/style-layout/tab_panels_with_amp-selector/ and the images are placeholder images.

- the first example works fine, since we are using the normal tabs.
- the second example works fine if you use the tabs, but the buttons won't work.
- the last example doesn't work at all.

note that on the second example, if you use the tabs first, and then the buttons afterwards, you will still see the images rendered at the tab press.
